### ISSUE TITLE
xaml intellisense

### ISSUE THREAD

**Issue by wku-trident** [DavidBurela/DevSuperPowers-Xamarin-Demo#1]
Hi David, 

please advise on how you got the intellisense for xaml?
Thanks in advance!

**Comment by DavidBurela** [comment]
I used resharper. I highly recommend it.
But I have heard there is a way to add a Visual Studio extension to get it http://www.cazzulino.com/mobileessentials.html

**Comment by wku-trident** [comment]
thanks for your reply. I tried resharper but it does not seems to work for
me.
Had also tried the vs ext that you mentioned but your resharper seems much
more productive

**Comment by DavidBurela** [comment]
I just tried disabling resharper and the intellisense goes away, enable and it comes back. So it is definitely Resharper doing it for me.
![xamarin intellisense](https://cloud.githubusercontent.com/assets/1320729/7362113/aa43aece-eda7-11e4-8d88-7cfb24a5a410.png)

Be sure you are using a Portable Xamarin project, and not the Shared project. The Shared one just breaks all intellisense

**Comment by wku-trident** [comment]
yes, I am using the PCL. Will try to figure out this resharper issue.

Hope to watch more Xamarin.Forms videos from you, thanks in advance!
